### ISSUE TITLE
onChange value + inputMask

### ISSUE THREAD

**Issue by rubgithub** [codegrue/card_settings#132]
The onChange value doesn't match when applying inputMask, for example when typing: 19216812
```
 inputMask: '000.000.000.000',
 onChanged: (value) {
                    print('onChanged $value');
```

It prints:

> I/flutter (30176): onChanged 1
> I/flutter (30176): onChanged 19
> I/flutter (30176): onChanged 192
> I/flutter (30176): onChanged 1921 (ERROR)
> I/flutter (30176): onChanged 192.16
> I/flutter (30176): onChanged 192.168
> I/flutter (30176): onChanged 192.1681 (ERROR)
> I/flutter (30176): onChanged 192.168.12

**Comment by codegrue** [comment]
Which widget is this on?

**Comment by rubgithub** [comment]
> Which widget is this on?

`CardSettingsText`

**Comment by codegrue** [comment]
Seems to be an issue with Flutter's TextField... the onChanged hander gets called before the controller is updated. Looking for a workaround.

**Comment by codegrue** [comment]
Fixed in 1.15.0
